problems/secondSmallest.js

```diff
@@ -6,8 +6,19 @@
 * @returns {number} - Returns the second smallest number.
 */
 
-function secondSmallest() {
-  
+const secondSmallest = (arr) => {
+  let smallest = Infinity;
+  let secondSmallest = Infinity;
+  for (let i = 0; i < arr.length; i++) {
+      const el = arr[i];
+      if (el < smallest) {
+          secondSmallest = smallest;
+          smallest = el;
+      } else if (el > smallest && el < secondSmallest) {
```

Comment:
No need for left side of && because if el was less than smallest, we would have entered the first if statement. Also, this could effect if we had a duplicate of the smallest number.